src/ORM/Behavior/CounterCacheBehavior.php

```diff
@@ -227,7 +227,7 @@ protected function _processAssociation(Event $event, EntityInterface $entity, As
                 continue;
             }
 
-            if (!is_string($config) && is_callable($config)) {
+            if (is_callable($config)) {
```

Comment:
This string check was to prevent global functions being used as callbacks.

Reply:
I see. That makes sense, I can revert. It was just weird that the method never allowed for any string input. The IDE complained here.

Reply:
I fixed it with an explicit exception inside the if block. I think that is cleaner.